frontend/layouts/default.vue

```diff
@@ -13,7 +15,7 @@ onMountedWithLoading(() => { /* nothing to do */ }, 'defaultLayout.onMountedWith
     >
       <main
         class="px-3 md:px-4 w-full lg:w-10 xl:w-8 mx-auto"
-        style="min-height: calc(100vh - 9.25rem - 4px);"
+        style="min-height: calc(100vh - 10.5rem - 4px);"
```

Comment:
Should this calc be pulled out into a variable somewhere? Seems error prone to remember to update it here and `StandardContent`

Reply:
GC - done.